frontend/src/lib/components/DateFilter/DateFilterExperiment.scss

```diff
@@ -0,0 +1,3 @@
+.DateFilter {
+    padding: 0.5rem;
+}
```

Comment:
`.DateFilter` [doesn't really belong](https://github.com/PostHog/posthog.com/pull/3703/files#diff-0c9e778f413523ae3326b1d530e860e1616e9dc6cd820db85f49940288496a47R41) in `DateFilterExperiment.scss` 🤔